Tests/ApolloCacheDependentTests/FetchQueryTests.swift

```diff
@@ -43,6 +43,48 @@ class FetchQueryTests: XCTestCase {
     }
   }
   
+  func testReturnCacheDataAndFetch() throws {
+    let query = HeroNameQuery()
+    
+    let initialRecords: RecordSet = [
+      "QUERY_ROOT": ["hero": Reference(key: "hero")],
+      "hero": [
+        "name": "R2-D2",
+        "__typename": "Droid",
+      ]
+    ]
+    
+    withCache(initialRecords: initialRecords) { cache in
+      let store = ApolloStore(cache: cache)
+      
+      let networkTransport = MockNetworkTransport(body: [
+        "data": [
+          "hero": [
+            "name": "Luke Skywalker",
+            "__typename": "Human"
+          ]
+        ]
+        ])
+      
+      let client = ApolloClient(networkTransport: networkTransport, store: store)
+      
+      let expectation = self.expectation(description: "Fetching query")
+      
+      client.fetch(query: query, cachePolicy: .fetchIgnoringCacheData) { (result, error) in
```

Comment:
It seems this isn't testing what you intended to test, because the cache policy is `.fetchIgnoringCacheData`.